README.md

```diff
@@ -8,7 +8,7 @@
 ![](https://github.com/fastify/fastify/workflows/package-manager-ci/badge.svg)
 ![](https://github.com/fastify/fastify/workflows/website/badge.svg)
 [![Known Vulnerabilities](https://snyk.io/test/github/fastify/fastify/badge.svg)](https://snyk.io/test/github/fastify/fastify)
-[![Coverage Status](https://coveralls.io/repos/github/fastify/fastify/badge.svg?branch=master)](https://coveralls.io/github/fastify/fastify?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/fastify/fastify/badge.svg?branch=HEAD)](https://coveralls.io/github/fastify/fastify?branch=HEAD)
```

Comment:
This image doesn't work, the `HEAD` trick seems to be broken with `coveralls.io` domain